utils/oscap.8

```diff
@@ -75,6 +75,11 @@ You may specify OVAL Definition files as the last parameter, XCCDF evaluation wi
 Select a particular profile from XCCDF document.
 .RE
 .TP
+\fB\-\-rule RULE\fR
+.RS
+Select a particular rule from XCCDF document. Only this rule will be evaluated.
```

Comment:
`(default)` profile is a profile and also can specify Values. As far as the API is concerned it is a proper profile.

I agree that mentioning that --profile affects values that are used with the rules makes sense.